src/client/game.cpp

```diff
@@ -623,15 +622,15 @@ struct GameRunData {
 	u16 new_playeritem;
 	PointedThing pointed_old;
 	bool digging;
-	bool ldown_for_dig;
+	bool btn_down_for_dig;
 	bool dig_instantly;
 	bool digging_blocked;
-	bool left_punch;
+	bool punch;
```

Comment:
`punching` would be consistent with `digging` - its counterpart.

Reply:
You're right, will do.

Reply:
Done